integrations/java/openhouse-java-runtime/src/main/java/com/linkedin/openhouse/javaclient/exception/WebClientResponseWithMessageException.java

```diff
@@ -8,21 +8,37 @@
  *
  * <p>In this class we override the {@link #getMessage()} behavior, to append server response if it
  * exists. It doesn't change status code (ex: forbidden) or status int (ex. 403)
+ *
+ * <p>An exception thrown in Openhouse clients to indicate an error in a tables API request, acting
+ * as a wrapper around WebClientResponseException.
  */
-public class WebClientResponseWithMessageException extends WebClientResponseException {
-  public WebClientResponseWithMessageException(WebClientResponseException e) {
-    super(
+public class WebClientResponseWithMessageException extends WebClientWithMessageException {
+  private final WebClientResponseException responseException;
+
+  /**
+   * We want the response exception to go through the legacy constructor in {@link
+   * WebClientResponseException} because the HttpRequest value is explicitly set to null in order to
+   * have the message rewritten to not expose unnecessary data.
+   */
```

Comment:
Is this relevant now that the wrapped response exception is hidden (`private`)? Before it made sense since the exception's values would be directly exposed, but now the comment about "not expose unnecessary data" seems nonsensical since the exception isn't exposed at all.

Reply:
the comment is moreso referring to the error message constructed from the exception, since if we do not pass a `null` value to the `HttpRequest` field as a part of `WebClientResponseException`, the error message will contain the method and URI contained from the request (eg: `<status code> <status type> from <request method> <request URI>`).

Reply:
Got it thanks!